apps/new-tab/types/chrome-18n.d.ts

```
@@ -0,0 +1,31 @@
+    i18n: {
+      getMessage: (
+        message: MessageKeys,
+        substitutions?: string | string[]
+      ) => string;
+    };
+    bookmarks: any | undefined;
+    storage: any | undefined;
+  };
+}
+
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)